django_extensions/management/commands/runserver_plus.py

```diff
@@ -387,37 +387,38 @@ def make_environ(self):
         )
 
     @classmethod
-    def _create_path_with_extension_from(cls, file_path, extension):
-        dir_path, cert_file = os.path.split(file_path)
-        if not dir_path:
-            dir_path = os.getcwd()
-        file_name, _ = os.path.splitext(cert_file)
-        return os.path.join(dir_path, file_name + "." + extension)
+    def determine_ssl_files_paths(cls, options):
+        key_file_path = options.get('key_file_path') or ""
+        cert_path = options.get('cert_path') or ""
+        cert_file = cls._determine_path_for_file(cert_path, key_file_path, cls.DEFAULT_CRT_EXTENSION)
+        key_file = cls._determine_path_for_file(key_file_path, cert_path, cls.DEFAULT_KEY_EXTENSION)
+        return cert_file, key_file
+
+    DEFAULT_CRT_EXTENSION = ".crt"
```

Comment:
Could you move the `DEFAULT_CRT_EXTENSION` and `DEFAULT_KEY_EXTENSION` to top of the class (under `requires_system_checks`) ?

Reply:
Ok